src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -574,6 +588,12 @@ function _validate_access_keys(access_key, secret_key) {
         throw_cli_error(ManageCLIError.MissingAccountAccessKeyFlag);
     }
 
+    // checking access_key and secret_key do not contain only whitespace
```

Comment:
I think the comment should match what you check in the function - validate_no_leading_whitespace.

Reply:
sure, thanks